frontend/app/controllers/MemberOnlyContent.scala

```diff
@@ -0,0 +1,41 @@
+package controllers
+
+import com.gu.contentapi.client.model.v1.{MembershipTier => ContentAccess}
+import com.gu.i18n.CountryGroup.UK
+import com.netaporter.uri.Uri
+import com.netaporter.uri.dsl._
+import com.typesafe.scalalogging.LazyLogging
+import configuration.{Config, CopyConfig}
+import model._
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import play.api.mvc._
+import services.{GuardianContentService, _}
+import tracking.ActivityTracking
+import views.support.PageInfo
+
+object MemberOnlyContent extends Controller with ActivityTracking
+                                 with LazyLogging {
+
+  val contentApiService = GuardianContentService
+
+  def membershipContent(referringContent: String) = NoCacheAction.async { implicit request =>
```

Comment:
I need to have a bit more thought about whether we could actually cache this, which would obviously be preferable.